### PR TITLE
Fixed legacy "view my shop" link when multistore is not used

### LINKED ISSUE #23804: [BO] Link View my shop is not displayed on Symfony pages
#### Describe the bug

On Symfony pages, The link `View my Shop` is not displayed

#### Expected behavior

Link is always displayed on BO

#### Steps to Reproduce

1. Go to BO
2. Sign in 
3. Go to a symfony page (Orders for example)
4. See error => NO link on the page

**Screenshots**
image

#### Additional information

* PrestaShop version: develop branch
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | The "view my shop" in the legacy header was hidden when multistore was not used. This PR fixes it.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23804
| How to test?      |  - Go to BO<br>- Sign in<br>- Go to a symfony page (Orders for example)<br>-  => link should appear (see issue)
| Possible impacts? |

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23818)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]

**Comment by matthieu-rolland** [comment]
gotta fix unit tests 😓  I'm on it !

update: done.

File: admin-dev/themes/new-theme/template/layout.tpl [hunk 1]
```diff
@@ -59,7 +59,7 @@
         </div>
       {/if}
 
-      {if !isset($hideLegacyStoreContextSelector)}
+      {if !$hideLegacyStoreContextSelector || !isset($hideLegacyStoreContextSelector)}
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/template/layout.tpl]:
```suggestion
      {if !isset($hideLegacyStoreContextSelector) || !$hideLegacyStoreContextSelector}
```

**Review by Progi1984 — COMMENTED** [review]
Some comments

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Doesn't add the waiting for QA label until PHPUnit Tests are done

**Comment by aDamelincourt** [comment]
Thanks @matthieu-rolland 
✔️  Check by QA
I can see link on legacy pages 
image

I can see link on Symfony pages
image

**Comment by Progi1984** [comment]
Thanks @matthieu-rolland & @aDamelincourt

**Pull request merged**